src/scripts/main.js

```diff
@@ -1 +1,11 @@
 'use strict';
+
+document.querySelectorAll('.gallery__thumb').forEach(function (thumbnail) {
+  thumbnail.addEventListener('click', function (eve) {
+    eve.preventDefault();
+
+    const largeImg = document.getElementById('largeImg');
+
+    largeImg.src = eve.target.src;
```

Comment:
The line `largeImg.src = eve.target.src;` assumes that the event target is the image element itself. If the thumbnail is wrapped in another element (e.g., a link or a div), `eve.target` might not be the image. Consider using `eve.currentTarget.querySelector('img')` or a similar approach to ensure you are getting the correct image source.